virttest/utils_selinux.py

```diff
@@ -278,6 +278,26 @@ def set_context_of_file(filename, context, selinux_force=False):
     logging.debug("Set context of %s success.", filename)
 
 
+def check_context_of_file(filename, label, selinux_force=False):
```

Comment:
var 'label' not used, do you really need this var?

Reply:
yes.. would be used in this function